### PR TITLE
Fix reshare with user activity message

### PR DESCRIPTION
**Pull request by nickvergessen**
Found while reviewing #3844

### CODE REVIEW

File: apps/files_sharing/lib/Activity/Providers/Users.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ public function parseLongVersion(IEvent $event) {
 		} else if ($event->getSubject() === self::SUBJECT_UNSHARED_USER_SELF) {
 			$subject = $this->l->t('You removed {user} from {file}');
 		} else if ($event->getSubject() === self::SUBJECT_RESHARED_USER_BY) {
-			$subject = $this->l->t('{actor} removed {user} from {file}');
+			$subject = $this->l->t('{actor} shared {user} with {file}');
```
Comment (LukasReschke) [inline apps/files_sharing/lib/Activity/Providers/Users.php]: Shouldn't that be "{actor} shared {file} with {user}"?

**Review by rullzer — APPROVED** [review]
:hear_no_evil:

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
I squashed both commits and fixe the sign-off failure 😉

**Comment by MorrisJobke** [comment]
@karlitschek I opened a backport in #3981 ;)

**Comment by karlitschek** [comment]
:+1:

**Pull request merged**